### PR TITLE
fix: handle requests with no account number

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -39,6 +41,12 @@ async function getAccountSnapshot(accountNumber) {
 app.use(async (request, response, next) => {
   if (request.path !== '/') {
     const accountNumber = request.path.split('/')[2];
```
Comment (steveoh) [inline index.js:43]: **question**: could this two throw an index out of bounds error? what is the path going to be ordinarilly?

**Review by steveoh — APPROVED** [review]
looks good. just a question

File: index.js (hunk 1 above)
Comment (stdavis) [reply index.js:43]: No, JS just returned undefined if you request an index that doesn't exist:
image

The path is normally `/v2/<account number>/arcgis/...`

Comment (steveoh) [reply index.js:43]: so then account number if it wasn't supplied comes in like `/v2/arcgis/` so account number wouldn't be a number then?

**Pull request merged**